Formula/sheldon.rb

```diff
@@ -28,7 +28,21 @@ class Sheldon < Formula
   end
 
   def install
-    system "cargo", "install", *std_cargo_args
+    # Ensure the declared `openssl@1.1` dependency will be picked up.
+    # https://docs.rs/openssl/latest/openssl/#manual
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
+    ENV["OPENSSL_NO_VENDOR"] = "1"
+
+    # Read the default flags from `Cargo.toml` so we can remove the `vendored-libgit2` feature.
+    cargo_toml = (buildpath/"Cargo.toml").read
+    cargo_option_regex = /default\s*=\s*(\[.+?\])/m
+    cargo_options = JSON.parse(cargo_toml[cargo_option_regex, 1].sub(",\n]", "]"))
+    cargo_options.delete("vendored-libgit2")
```

Comment:
this package is a little different: https://github.com/rossmacarthur/sheldon/blob/0.7.2/Cargo.toml#L37

an in-replace should work: 

```ruby
inreplace "Cargo.toml", /features = \["vendored-libgit2"\]/, "features = []"
```